packages/clients/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shinami/clients",
-  "version": "0.9.0",
+  "version": "0.10.0",
```

Comment:
Let's do `0.9.1` since it's non-breaking.

Reply:
Would this not qualify as a minor version bump? Adding functionality in backwards compatible (non breaking) manner: https://semver.org/#semantic-versioning-200

Reply:
Normally yes, new features without breaking chagnes should be introduced with minor version bumps. The special case here is we are still on `0.*`, in which case even minor version bumps are considered breaking by [npm](https://semver.npmjs.com/). E.g.  `0.10.0` is NOT compatible with `^0.9.0`.